src/dmd/mtype.d

```diff
@@ -2175,6 +2190,18 @@ extern (C++) abstract class Type : RootObject
         return 0;
     }
 
+    /***************************************
+     * Return MOD bits matching this type to wild parameter type (tprm) without resolving alias this.
+     */
+    final ubyte deduceWildWithoutAliasThis(Type t, bool isRef)
```

Comment:
You may want to explicitly return `MOD`.  See also https://github.com/dlang/dmd/pull/8495

Reply:
Done